<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminmenu</class>
 <widget class="QWidget" name="adminmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>adminmenu</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-4</y>
     <width>1280</width>
     <height>907</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget{
	
	border-image: url(:/LIBPRO/pattern-5_blur.png);
}

QPushButton {	
	font-family: &quot;Silom&quot;;
	color: #145acc;
	border: 2px solid grey;	
	border-radius: 15px;
	background-color: qlineargradient(spread:reflect, x1:0.669951, y1:0.727273, x2:0, y2:0, stop:0 rgba(168, 237, 205, 255), stop:0.492611 rgba(255, 255, 255, 255));

}


QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(212, 236, 215));
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}


QLineEdit {
	border: 1px solid grey;
	border-radius: 7px;
    padding: 10px;
    width: 200px;
    height: 150px;
}

QLabel {
	color: #646568;
	font-size: 14px;
	font-weight: bold;
	font-family: &quot;Bangla Sangam MN&quot;;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="accountDetail">
    <widget class="QCommandLinkButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>90</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-radius: 15px;
background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>search_icon.png</normaloff>search_icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLine">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>691</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Search account</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>1150</width>
       <height>661</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1150</width>
        <height>661</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkUser">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>User</string>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkLib">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>100</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Librarian</string>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkAd">
     <property name="geometry">
      <rect>
       <x>1110</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>16</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>20</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="LIBPRO.qrc">
       <normaloff>:/LIBPRO/Alecive-Flatwoken-Apps-Help-Info.ico</normaloff>:/LIBPRO/Alecive-Flatwoken-Apps-Help-Info.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>power-button-md.png</normaloff>power-button-md.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="addUser">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="LIBPRO.qrc">
       <normaloff>:/LIBPRO/Ampeross-Qetto-2-Add-contact.ico</normaloff>:/LIBPRO/Ampeross-Qetto-2-Add-contact.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LIBPRO.qrc"/>
 </resources>
 <connections/>
</ui>
